### ISSUE TITLE
Sass convert removes meaningful parentheses in expression

### ISSUE THREAD

**Issue by joshprice** [sass/sass#13]
I just installed sass 3.0.21 and ran sass-convert over all my sass using the following command:

```
sass-convert --in-place --from sass2 <file>.sass
```

There was an issue when it converted the following:

```
:width #{!blueprint_container_size - (!padding + !padding) - 2px}
```

into this:

```
width: #{$blueprint_container_size - $padding + $padding - 2px}
```

Note the lack of parentheses changing the expression completely.

Not sure why sass-convert would arbitrarily remove parentheses here, as other expressions converted in the same run had correctly preserved parentheses.

**Comment by nex3** [comment]
`a - b + c - d` is equivalent to `a - (b + c) - d`. Sass doesn't keep track internally of where parentheses were; it just keeps track of the structure of the expression. Then when converting back to source code, it uses the simplest form with the fewest parentheses.

**Comment by joshprice** [comment]
Expanding `a - (b + c) - d` should give `a - b - c - d`. (Note the  `- c` not `+ c`) It's not possible to drop the parentheses here without expanding the expression correctly.

**Comment by nex3** [comment]
Ah, you're right. I've re-opened the issue.

**Comment by nex3** [comment]
Merge remote branch 'haml/stable' into stable

Closed by 8a9871f3ab0cec94e9c8ab32d290ab13677b1a78

**Comment by nex3** [comment]
This was actually closed by 683b7c2ed96cc89d250ab6d4bf8091326a0814bf

**Comment by joshprice** [comment]
Nice work! Thanks.
